web/src/index.js

```diff
@@ -58,20 +60,25 @@ const App = () => {
     }).catch((error) => {
       setAuthorized(false);
     });
-  },[token]); // monitor token change then trigger this
+  },[]); // monitor token change then trigger this
```

Comment:
Looks good for using react hook and state. 👍 
We need to fetch token from the query string first and if null then fetch the local cookie variable, right? Store the token into Cookie.
TODO:  Update setUser response data format.
I modified user API SEPC, will deploy it to the staging site (TBD), need to update setUser response data format.
https://docs.google.com/document/d/1qnv3KIdgGcTaHXfHtPJda9kDZ5uwqFUT6voH-8nRWG0/edit#heading=h.f2sa6ucbmwhr